src/reducers/index.js

```diff
@@ -77,15 +76,9 @@ function getInitialState() {
     columnSettings: {
       defaultColumnWidth: 100,
       bufferColCount: undefined,
-      fixedColumnsCount: 0,
-      fixedRightColumnsCount: 0,
-      scrollableColumnsCount: 0,
```

Comment:
Why move these away from `columnSettings`?

Reply:
What we are supposed to keep in `columnSettings` ? i thought it contains settings that client has passed to FDT as props. whereas these are now going to  be derived data.